packages/slack-app/src/routes/learning-collector/index.ts

```diff
@@ -193,7 +193,7 @@ async function postMessageWithReactions(
 
   const chatPostMessageResponse = await client.chat.postMessage(message);
   for (const reaction of reactions) {
-    client.reactions.add({
+    await client.reactions.add({
```

Comment:
You probably shouldn't await inside a for loop. Instead just push the Promise to an array and then await the whole array after the for loop. That way each request can run in parallell instead of waiting for one at a time.